dev/DatePicker/DatePicker_themeresources.xaml

```diff
@@ -384,31 +388,35 @@
                                 <RowDefinition Height="*" />
                                 <RowDefinition Height="Auto" />
                             </Grid.RowDefinitions>
-                            <Grid x:Name="PickerHostGrid">
+                            <Grid>
```

Comment:
Is the new grid needed? I think we could avoid it by setting column span on the HighlightRect to 5

Reply:
HighlightRect needs to draw *over* the PickerGrid because it's actually drawing opaque content to obscure what the PickerGrid drew with the inverted colors.

Reply:
Children draw over there parents and their older siblings though, right?

Reply:
Yes but the Panels that host the children are conspicuously absent from the template because they are Append()-ed from code behind in the picker code. :( So no matter what in that single grid you can't draw *over* those panels from the template.

Reply:
> Yes but the Panels that host the children are conspicuously absent from the template because they are Append()-ed from code behind in the picker code. :( So no matter what in that single grid you can't draw over those panels from the template.

Is that something that could be changed with WinUI 3 or do you think it would very hard to change the behavior here? Having controls appending elements in their code behind does not sound ideal, especially for templated controls so it might be good to change this.

Reply:
I saw mention of Acrylic Accent Colour brushes, could on of those be used for the selection rectangle?